product_docs/docs/tde/15/initdb_tde_options.mdx

```diff
@@ -21,9 +21,9 @@ Provides the wrapping/encryption command to protect the data encryption key.
 
 `<command>` is customizable, but it must contain the placeholder `%p`. See [Wrapping commands](secure_key/#configuring-a-wrapping-and-unwrapping-command) for examples and information. See [Securing the data encryption key](secure_key) for an overview of available wrapping mechanisms.
 
-If not specified, TDE falls back on the environment variable `PGDATAKEYWRAPCMD`.
+If you don't specify a command, TDE falls back on the environment variable `PGDATAKEYWRAPCMD`.
```

Comment:
Please check this edit. I wasn't sure if it was the command or the %p that wasn't specified.